MrtCore/console_unpackaged_app/console_unpackaged_app.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <windows.h>
+#include <iostream>
+
+#include "winrt\Windows.Foundation.h"
+#include "winrt\Windows.Foundation.Collections.h"
+#include "winrt\Microsoft.ApplicationModel.Resources.h"
+
+#include <MddBootstrap.h>
+
+using namespace winrt;
+using namespace winrt::Microsoft::ApplicationModel::Resources;
+
+HRESULT MddBootstrapInitializeWrapper()

```

Comment:
Maybe rename this so there's no confusion that this is app code, rather than mdd code?  Like call it `LoadProjectReunion` or something.